dv/verilator/memutil/cpp/verilator_memutil.cc

```diff
@@ -410,7 +426,8 @@ bool VerilatorMemUtil::MemWrite(const MemArea &m, const std::string &filepath,
 }
 
 bool VerilatorMemUtil::WriteElfToMem(const svScope &scope,
-                                     const std::string &filepath) {
+                                     const std::string &filepath,
+                                     const size_t size_bytes) {
```

Comment:
```suggestion
                                     const size_t size_byte) {
```

To match `VerilatorSimCtrl::FileSize()`

And I'd remove the `const` for pass-by-value arguments, it doesn't add a lot of meaning, but typing.